src/main/java/conditions/BooleanRegexCondition.java

```diff
@@ -51,9 +51,6 @@ public Boolean getReverse() {
     @Override
     public Boolean apply( Map<String, String> row ) {
         String o = row.get( column );
-        if ( StringUtils.isBlank( o ) ) {
```

Comment:
`Cached.getInsensitiveMatcherForString` includes a `@NotNull` annotation for the `targetString` param. This section catches the case where `o` is null, but without this check it becomes possible to pass a null value to `getInsensitiveMatcherForString`, which is illegal. So you probably either want to keep this or replace it will an explicit null check if you want to allow empty strings.

Reply:
Ooh, ok. Will fix.

Reply:
Now explicitly converts `null` to `""`.